packages/react-router/lib/types/internal-export/route-data.ts

```diff
@@ -1,12 +1,12 @@
 import type {
```

Comment:
Anything directly exported from `internal-export.ts` now lives in the `types/internal-export/` folder